community/urls.py

```diff
@@ -24,8 +44,61 @@ def get_index():
     return None
 
 
-urlpatterns = [
+def get_all_contributors():
+    # This function needs to return an iterable of dictionaries. Dictionaries
```

Comment:
We do not need this comment repeated everywhere.  The code is ridiculously simple and obvious.